tests/test_benchmark.py

```diff
@@ -377,6 +446,14 @@ def run_single_topology_benchmarks(
             initial_state.ligand_idxs,
         )
 
+        if host_config.num_water_atoms < initial_state.x0.shape[0] - st.get_num_atoms():
```

Comment:
nit: maybe include a comment here to explain what this condition means?

Reply:
Added comment and made the statement a bit clearer: https://github.com/proteneer/timemachine/pull/1296/commits/ceb798d4ad739590cca23d2d4142cb6347734a4d